File: common/src/main/java/br/com/gustavoterras/foo/lib/common/CommonActivity.java
package br.com.gustavoterras.foo.lib.common;

import android.databinding.DataBindingUtil;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class CommonActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        DataBindingUtil.setContentView(this, R.layout.activity_common);
    }
}

File: common/src/main/java/br/com/gustavoterras/foo/lib/common/CommonActivity.kt
package br.com.gustavoterras.foo.lib.common

import android.databinding.DataBindingUtil
import android.databinding.ViewDataBinding
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class CommonActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        DataBindingUtil.setContentView<ViewDataBinding>(this, R.layout.activity_common)
    }
}
